File: samples/src/main/java/jvmMain/java/safe/AssetBalance.java
package jvmMain.java.safe;

import jvmMain.java.Common;
import jvmMain.kotlin.Config;
import one.mixin.bot.safe.OutputKt;

import java.util.ArrayList;

public class AssetBalance {

    public static void main(String[] args) {
        ArrayList<String> members = new ArrayList<>();
        members.add(Config.BOT_USER_ID);
        String assetBalance = OutputKt.assetBalance(Common.botClient, Common.Token.CNB.getAssetId(), members);
        System.out.println("assetBalance: " + assetBalance);
    }
}

File: samples/src/main/java/jvmMain/kotlin/safe/AssetBalance.kt
package jvmMain.kotlin.safe

import jvmMain.kotlin.Token
import jvmMain.kotlin.botClient
import one.mixin.bot.safe.assetBalance

fun main() {
    val balance = assetBalance(botClient, Token.CNB.assetId)
    println("balance: $balance")
}